projects/ion/src/styles/fonts/index.scss

```diff
@@ -0,0 +1,59 @@
+@import '../themes/index.scss';
+
+@font-face {
+  font-family: 'Source Sans Pro';
+  src: url('./SourceSansPro-Regular.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: 'Source Sans Pro';
+  src: url('./SourceSansPro-Regular.ttf');
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: 'Source Sans Pro';
+  src: url('./SourceSansPro-SemiBold.ttf');
+  font-weight: 600;
+}
+
+@font-face {
+  font-family: 'Source Sans Pro';
+  src: url('./SourceSansPro-Bold.ttf');
+  font-weight: 700;
+}
+
+@mixin font-size-12 {
```

Comment:
I believe that this names with numbers not is the better way, because if the font size change the number will be incorrect. What do you think change to sm, md, lg, xlg...?